docs/source/packaging/deb.rst

```diff
@@ -0,0 +1,75 @@
+==========================
+MOTECH packaging using DEB
+==========================
+
+This is a description of building process for DEB packages on Ubuntu
+
+Requirements
+============
+
+* Apache Tomcat 7.0 or greater, can be installed with apt-get command
+
+.. code-block:: bash
+
+    sudo apt-get install tomcat7
+
+* Java 1.8 or greater, can be installed with apt-get command
+
+.. code-block:: bash
+
+    sudo add-apt-repository ppa:webupd8team/java
+    sudo apt-get update
+    sudo apt-get install oracle-java8-installer
+
+* Apache Maven 3 (needed only for building package)
```

Comment:
We can refer to the developer setup doc for such things. But don't we need things like lintian or build-essential to build the deb? I doubt those are mentioned in dev setup doc